### ISSUE TITLE
Ministation make-ready

### ISSUE THREAD

**Issue by octareenroon91** [Zaers/-tg-station#215]
Continue the efforts of #210 to make Ministation fun to play.

**Comment by Asd396** [comment]
Might as well repost my ideas
* Restore atmos and other functionality that is absent in mini
* Restore missing jobs, even if for example atmos techs would basically be engineers with different jumpsuits. Heads are probably going to remain how they are though.
* Give the AI an proper core. I was thinking I might move main telecommunications to a restored version of the abandoned satellite or Ruskie, accessible with a gated teleporter.

**Comment by Asd396** [comment]
Oh, right, and should we consider getting some sort of map changing/voting script first?

**Comment by octareenroon91** [comment]
I'd like to have a working map to switch to first.  I can toss it on sparestation if nothing else.

**Comment by Asd396** [comment]
Seems like /tg/ actually has map rotation too so we don't need to port it from /vg/. See https://github.com/tgstation/-tg-station/pull/12089

**Comment by octareenroon91** [comment]
Yes, we're running on linux. Bats can probably be recoded into shell scripts.

**Comment by Asd396** [comment]
I'm working on mini right now. Would you like to port map rotation and any other bots?

**Comment by octareenroon91** [comment]
See if you can figure out how they automate changelog processing. There's a chance we'd have to move off Zaers for some features.

**Comment by octareenroon91** [comment]
I can't remember if I pushed my very boxy atmos...

I remember planning to give the garden the good hydroponics, adding some maint paths or something east of EVA, and adding a robotics
